runelite-client/src/main/java/net/runelite/client/plugins/screenshot/ScreenshotPlugin.java

```diff
@@ -717,6 +729,19 @@ static String parseBAHighGambleWidget(final String text)
 		return "High Gamble(count not found)";
 	}
 
+	/**Parses the combat achievement success chat message into a string for filename. ...*/
```

Comment:
Use full javadoc formatting here.
```suggestion
	/**
	 * Parses a combat achievement success chat message into a filename-safe string.
	 *
	 * @param text A received chat message which may or may not be from completing a combat achievement.
	 * @return A formatted string of the achieved combat task tier and name, or the empty string if the passed message
	 *         is not a combat achievement completion message.
	 */
```